client/src/components/common/index.js

```diff
@@ -3,6 +3,15 @@ import Loader from './Loader';
 import Card from './Card';
 import FilterComponent from './Filter';
 import Avalibility from './availabilityTable';
+import AvailableityTime from './AvailabilityTime';
```

Comment:
you can export the file form the index file at the common folder 
```suggestion
import AvailableityTime from './AvailabilityTime';
import {AvailableityTime, Avalibility} from '../../common'
```